PWGDQ/Tasks/tableReader.cxx

```diff
@@ -999,5 +999,17 @@ void DefineHistograms(HistogramManager* histMan, TString histClasses)
     if (classStr.Contains("DileptonHadronCorrelation")) {
       dqhistograms::DefineHistograms(histMan, objArray->At(iclass)->GetName(), "dilepton-hadron-correlation");
     }
+
+    if (classStr.Contains("PIDCalib")) {
```

Comment:
I see that you add here the histogram classes, but you did not enabled these histogram classes in the init() and I also don't see where you are filling them in the process function. You still need to add some code?

Reply:
Hi Ionut, 
Now we moved the histogram to the track group, see above. we don't need anything  init().
Cheers
Xiaozhi